core/domain/value_generators_domain.py

```diff
@@ -61,9 +61,15 @@ def get_html_template(cls):
             os.getcwd(), feconf.VALUE_GENERATORS_DIR, 'templates',
             '%s.html' % cls.__name__))
 
-    def generate_value(self, *args, **kwargs):
+    # Since child classes of BaseValueGenerator can use
+    # the generate_value functions with different types
+    # of arguments, args, kwargs and return type is Any.
```

Comment:
```suggestion
    # Since child classes of BaseValueGenerator can use
    # the 'generate_value' function with different types
    # of arguments, 'args', 'kwargs' and return type are set to 'Any'.
```

Reply:
Done.